File: MOC Develop SQL 2005/Labfiles/Lab07/Solution/cs/ManagedUtilities2/GetLongDate.cs
using System;
using System.Data.Sql;
using System.Data.SqlTypes;

namespace DateUtilities
{
	public partial class UserDefinedFunctions
	{
		[SqlFunction(Name = "GetLongDate",
					 DataAccess = DataAccessKind.None)]
		public static SqlString GetLongDate(SqlDateTime DateVal)
		{
			return DateVal.Value.ToLongDateString();
		}
	};
}

File: MOC Develop SQL 2005/Labfiles/Lab07/Solution/vb/ManagedUtilities2/GetLongDate.vb
Imports System
Imports System.Data.Sql
Imports System.Data.SqlTypes

Partial Public Class UserDefinedFunctions
    <SqlFunction(Name:="GetLongDate", _
             DataAccess:=DataAccessKind.None)> _
    Public Shared Function GetLongDate _
     (ByVal DateVal As SqlDateTime) As SqlString
        Return DateVal.Value.ToLongDateString()
    End Function
End Class
